pages/docs/config/destinations.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Destinations
-date: "2021-10-13"
+date: "2021-01-18"
```

Comment:
```suggestion
date: "2022-01-18"
```

Reply:
I'm waiting until we're live to update these dates.